code/modules/events/shuttle_loan/shuttle_loan_datum.dm

```diff
@@ -213,3 +213,82 @@
 	for(var/i in 1 to 5)
 		var/turf/web_turf = pick_n_take(empty_shuttle_turfs)
 		new /obj/structure/spider/stickyweb(web_turf)
+
+#define DENT_WALL "dent"
+#define CHANGE_WALL "change"
+#define DISMANTLE_WALL "dismantle"
+
+#define BREAK_TILE "break"
+#define PLATING_TILE "plating"
+#define RUST_TILE "rust"
+
+/**
+ * A special shuttle loan situation enabled by the 'mail blocked' station trait.
+ * It sends back a lot of mail to the station, at the cost of wrecking the cargo shuttle a little.
+ */
+/datum/shuttle_loan_situation/mail_strike
+	sender = "Spinward Mail Workers Union"
+	announcement_text = "The Mail Workers Union wants to borrow your cargo shuttle to employ \"advanced union strike tactics\" with. Payment is strictly in mails."
+	bonus_points = 0
+	thanks_msg = "The cargo shuttle should return in five minutes."
+	shuttle_transit_text = "Nothing stops the mail."
+	logging_desc = "Shuttle full of shady mail"
+
+/datum/shuttle_loan_situation/mail_strike/spawn_items(list/spawn_list, list/empty_shuttle_turfs, list/blocked_shutte_turfs)
+	for(var/i in 1 to rand(7, 12))
+		var/turf/closed/wall/wall = pick_n_take(blocked_shutte_turfs)
+		if(!istype(wall))
+			continue
+		var/static/list/wall_bad_stuff = list(DENT_WALL = 85, CHANGE_WALL = 13, DISMANTLE_WALL = 2)
+		var/static/list/possible_new_walls = list(
+			/turf/closed/wall/mineral/sandstone,
+			/turf/closed/wall/mineral/wood,
+			/turf/closed/wall/mineral/iron,
+			/turf/closed/wall/metal_foam_base,
+			/turf/closed/wall/r_wall,
+		)
+		var/damage_done = pick_weight(wall_bad_stuff)
+		switch(damage_done)
+			if(DENT_WALL)
+				for(var/dent in 1 to rand(1, MAX_DENT_DECALS))
+					wall.add_dent(prob(90) ? WALL_DENT_SHOT : WALL_DENT_HIT)
+			if(CHANGE_WALL)
+				wall.ChangeTurf(pick(possible_new_walls - wall.type))
+				if(prob(25))
+					for(var/dent in 1 to rand(1, MAX_DENT_DECALS))
+						wall.add_dent(prob(90) ? WALL_DENT_SHOT : WALL_DENT_HIT)
+			if(DISMANTLE_WALL)
+				wall.dismantle_wall()
+
+	for(var/i in 1 to rand(7, 12))
+		var/turf/open/floor/floor = pick_n_take(empty_shuttle_turfs)
+		if(!istype(floor))
+			continue
+		var/static/list/floor_bad_stuff = list(BREAK_TILE = 65, PLATING_TILE = 25, RUST_TILE = 10)
+		var/damage_done = pick_weight(floor_bad_stuff)
+		switch(damage_done)
+			if(BREAK_TILE)
+				if(prob(50))
+					floor.break_tile()
+				else
+					floor.burn_tile()
+			if(PLATING_TILE)
+				if(prob(25))
+					floor.remove_tile()
+				else
+					floor.make_plating()
+			if(RUST_TILE)
+				floor.ChangeTurf(/turf/open/floor/plating/rust)
+	if(prob(25))
+		spawn_list += pick(/obj/effect/gibspawner/robot, /obj/effect/gibspawner/human)
+
+	for(var/i in 1 to rand(4, 7))
+		spawn_list += /obj/structure/closet/crate/mail/full/mail_strike
+
+#undef BREAK_TILE
+#undef PLATING_TILE
+#undef RUST_TILE
+
+#undef DENT_WALL
+#undef CHANGE_WALL
+#undef DISMANTLE_WALL
```

Comment:
Trailing newline

Reply:
It's already there, otherwise the linters CI would have thrown a warning.